### ISSUE TITLE
Expand desktop corpus to all ~4M desktop CrUX origins

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/httparchive.org#114]
- [x] extract the desktop origins from CrUX and convert to URLs (by appending a `/`)
- [x] load a CSV of the new URLs onto the HA test server (easy to do via the HA CDN)
- [x] [change the desktop crawl capacity](https://github.com/HTTPArchive/legacy.httparchive.org/commit/b386bf30b895fd646ccd8f837b55ee2957f2b162) to handle 4M URLs
- [x] copy the URLs mysql table to a mobile-specific `urlsmobile` table so we can have separate corpuses for desktop and mobile
- [x] [point the mobile crawl](https://github.com/HTTPArchive/legacy.httparchive.org/commit/28704d8049f9bfc6df5badd6b158e3e9450ad576) to the new `urlsmobile` table (1.3M URLs)
- [x] run `make pushall` to apply the config changes to desktop and mobile
- [x] in mysql run `truncate table urlsdev;` to clear out the old list of 1.3M URLs
- [x] run `php bulktest/importurls.php <csv-file> other` to load the 4M URLs into the urlsdev table

Query to extract the desktop origins from the latest CrUX release:

```sql
SELECT
  DISTINCT CONCAT(origin, '/') AS url
FROM
  `chrome-ux-report.all.201811`
WHERE
  form_factor.name = 'desktop'
```

Table of desktop URLs: https://bigquery.cloud.google.com/table/httparchive:urls.2018_12_15_desktop

**Comment by rviscomi** [comment]
```
mysql> select count(0) from urlsdev;
+----------+
| count(0) |
+----------+
|  3880557 |
+----------+
1 row in set (0.00 sec)

mysql> select count(0) from urlsmobile;
+----------+
| count(0) |
+----------+
|  1294054 |
+----------+
1 row in set (0.44 sec)
```

**Comment by rviscomi** [comment]
> 1. All the old URLs are dead and gone now? (for desktop)

Of the 1.3M desktop URLs we've been testing for ~6 months, 1M of them are still in the corpus. We considered including the remaining 300k URLs for consistency across crawls but opted not to for simplicity.

(we = myself, @igrigorik @pmeenan @paulcalvano)

> 2. Presumably you're doing the same thing for mobile?

Yes. We plan to roll that out next month.

> 3. @brendankenny said there weren't a lot of https urls in the old HA url list.. Whats the balance in this new URL list?

This list is 60% HTTPS:

```sql
SELECT
  SUM(IF(STARTS_WITH(url, 'https'), 1, 0)) / COUNT(0) AS pct_https
FROM
  `httparchive.urls.2018_12_15_desktop`
```

> 4. We thought the 1.5M run was barely finishing in under 2 weeks. Is there new WPT capacity being added to handle this load?

Yeah, we used to run each page 3 times on desktop, 3 on mobile, and 1 on Lighthouse. As of December 1 we're doing it 1 time on all 3 settings. That affords us the space to increase the desktop corpus. To afford the mobile increase, we're going to reduce the crawl frequency from 15 days to monthly. I wrote up a short summary / sanity check in [this doc](https://docs.google.com/document/d/1J8Bc3QmvrABLiNx9mV9rYx5jFBCW3f1mU1vcQcKsQj8/edit?usp=sharing).
